recipes/ansys-pythonnet/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ansys-pythonnet" %}
+{% set version = "3.1.0rc3" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
I've added the required change, thanks for pointing that out!